packages/terra-time-input/translations/fi-FI.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Looks like fr-FR and fr needs filled with mock translations

Reply:
I'll be getting a translation Jira logged for this component. I was waiting for feedback on this review before sending it out, but I think I have enough to get going. I'll get mocks added for these individual files in the mean time.

Reply:
Added. https://github.com/cerner/terra-core/pull/1027/commits/5073f9f2f68e92dc3b20b5a431b9e8f026b8b78e